airflow/operators/papermill_operator.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
not sure if there is any consensus on putting which operator where, but won't this belongs to contrib section?

Reply:
I think the idea is to drop the contrib section somewhere close in the future.